app/src/main/java/com/kamron/pogoiv/updater/AppUpdateLoader.java

```diff
@@ -19,18 +17,19 @@
     @Override
     public void run() {
         OkHttpClient httpClient = new OkHttpClient();
-        String apiEndpoint = "https://api.github.com/repos/farkam135/GoIV/releases";
+        String apiEndpoint = "https://api.github.com/repos/farkam135/GoIV/releases/latest";
         Request request = new Request.Builder()
                 .url(apiEndpoint)
                 .build();
 
         try {
             Response response = httpClient.newCall(request).execute();
-            JSONArray releaseInfo = new JSONArray(response.body().string());
-            JSONObject latestRelease = releaseInfo.getJSONObject(0);
-            JSONObject releaseAssets = latestRelease.getJSONArray("assets").getJSONObject(0);
+            JSONObject releaseInfo = new JSONObject(response.body().string());
+            JSONObject releaseAssets = releaseInfo.getJSONArray("assets").getJSONObject(0);
+            if(releaseAssets.getString("name").contains("Offline"))
```

Comment:
BTW, can you please reformat the code using Android Studio's formatter? I've reformatted stuff, we should keep it that way from now on.